data/CO/incentives.json

```diff
@@ -7544,6 +7544,75 @@
     },
     "low_income": "co-colorado-energy-office"
   },
+  {
+    "id": "CO-545",
+    "authority_type": "state",
+    "authority": "co-colorado-energy-office",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "heat_pump_air_conditioner_heater",
+    "program": "co_coloradoEnergyOffice_coloradoHeatPumpTaxCredits",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 499.95
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$499.95 discount for ENERGY STAR & AHRI certified system. Available through registered contractors.",
+      "es": "Descuento de $499.95 para sistemas certificados por ENERGY STAR y AHRI. Disponible a través de contratistas registrados."
+    },
+    "start_date": "2024-01-01",
+    "end_date": "2024-12-31"
+  },
+  {
+    "id": "CO-546",
+    "authority_type": "state",
+    "authority": "co-colorado-energy-office",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "geothermal_heating_installation",
+    "program": "co_coloradoEnergyOffice_coloradoHeatPumpTaxCredits",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 999.9
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$999.90 discount for ENERGY STAR & AHRI certified system. Available through registered contractors.",
```

Comment:
This value looks weird, but is the minimum a contractor is required to give the purchaser. see: https://energyoffice.colorado.gov/hptc